docs/generate_docs.py

```diff
@@ -290,6 +290,9 @@ def get_call_signature(obj):
                 default = f'"{default}"'
             else:
                 default = repr(default)
+
+            # Replace from repr because it can cause HTML errors in rendering
+            default = default.replace("<", "&lt;").replace(">", "&gt;")
```

Comment:
You should use [`html.escape`](https://docs.python.org/3/library/html.html#html.escape) instead.